envVariables.js

```diff
@@ -176,6 +176,21 @@ let enviromentVariables = {
 		message: 'Required admin access token',
 		optional: false,
 	},
+	USER_DELETE_ON_OFF: {
+		message: 'Enable/Disable User data deletion',
+		optional: true,
+		default: 'ON',
+	},
+	USER_DELETE_TOPIC: {
+		message: 'Required user data delete kafka topic',
+		optional: true,
+		default: 'user.create',
```

Comment:
@priyanka-TL This has to be user update i think, also optional false based on above key

Reply:
removing default value, hence no topic have default in this service, this can lead issues if that is conflicting. Added requireif check